backend/common/config.py

```
@@ -37,7 +37,7 @@ class Settings(BaseSettings):
     # Default to SQLite for local development only
     database_url: str = f"sqlite:///{BASE_DIR / 'app.db'}"
     secret_key: str = "super-secret-key"
```

Ruff (0.15.1)

[error] Possible hardcoded password assigned to: "secret_key"

(S105)